### DISCUSSION TITLE
Modifying `SkipRemainingOnFailure` while running pester?

### DISCUSSION THREAD

**Discussion by schittli** [pester/Pester#2259] [Q&A] [answered]
Good evening

I hope I have a simple question, but unfortunately I did not find the solution in many experiments:

Is it possible to change the runtime configuration of pester and set `SkipRemainingOnFailure`? 

Thanks a lot for any light in the dark 😃
Kind regards,
Thomas

**Comment by fflaten** [comment, marked as answer]
I assume you mean enable or modify it in the middle of a Pester-run, correct? If so the short answer is no.

`SkipRemainingOnFailure` is implemented as a plugin which loads different code based on the configuration provided at start. Though technically possible by calling internal API, it is not supported.

May I ask in what scenario you'd want this?
